README.md

```diff
@@ -119,6 +119,9 @@ Other languages are periodically pulled from [Transifex](https://www.transifex.c
 - Publish local changes and the tag to GitHub repo: `git push && git push origin vA.B.C`
 - Wait for the CI to upload the binaries to the draft release (a new one will be created if you haven't drafted one).
 - The `latest.yml, latest-mac.yml, latest-linux.yml` files on the release are used by the app to determine when an app update is available. Once a release is published, users should recieve the app update. See: https://www.electron.build/auto-update.
+- Notarize `.dmg` at Apple (context: [#1365](https://github.com/ipfs-shipyard/ipfs-desktop/issues/1211))
+    1. Download `.dmg` from `https://github.com/ipfs-shipyard/ipfs-desktop/releases/vA.B.C`
+    2. Run `node pkgs/macos/notarize-cli.js ./IPFS-Desktop-A.B.C.dmg`
```

Comment:
What do you do w/ the notarized dmg? Commit back to repo?

Reply:
Should adding and editing the .env file be here too?

Reply:
#### what to do with notarized .dmg?

We do nothing with notarized `.dmg`, just discard it for now. Goal is to inform apple that this is a legit binary blob, so it can give thumbs up when someone's macOS asks about it in the future.

Why? **We can't replace `.dmg` attached to GitHub Release with notarized version** because checksum in `latest-mac.yml` would no longer match and that would break autoupdates. 

But online notarization should still work: when macOS sees a new version of ipfs-desktop it will check for stapled notarization, won't find it, so it will fallback to asking Apple servers if running it is ok. Autoupdate requires internet connection anyway, so this non-stapled notarization should also work.

#### env

`notarize-cli` will inform user if env variables are missing